src/index.js

```diff
@@ -2,8 +2,12 @@ import {getQueriesForElement, prettyDOM} from 'dom-testing-library'
 
 export * from 'dom-testing-library'
 const mountedContainers = new Set()
-export const render = (Component, options) => {
-  const target = document.body.appendChild(document.createElement('div'))
+export const render = (Component, options = {}) => {
+  let target = document.body.appendChild(document.createElement('div'))
+
+  if (options.target) {
+    target = options.target
+  }
```

Comment:
Sorry to be a pest but can you destructure Target and use a default on that as well then use a spread on options?  Otherwise Target is being passed in twice.

Ie: `({target = document.appendChild(etc), ...options} = {})`

Reply:
it is all good mate, everyone has their own taste of code 😃